src/op.rs

```diff
@@ -199,6 +199,37 @@ where
         /// The offfset to add.
         offset: u64,
     },
+    /// Represents `DW_OP_const_type`.
+    TypedLiteral {
```

Comment:
The general approach taken by the expression evaluator is to have this enum unify the different representations of some thing into a single enumerator.  Extending this idea to the typed stack, all literals would have a type; just some would have the generic type.

Reply:
I guess we could do that